gallery/psget/get_psget_module.md

```diff
@@ -1,58 +0,0 @@
----
-ms.date:  06/12/2017
-contributor:  manikb
-ms.topic:  reference
-keywords:  gallery,powershell,cmdlet,psget
-title:  Get PowerShellGet Module
```

Comment:
This file should not be deleted. Effectively, this is the "Install" article. It should go under the "Getting Started" node in the TOC.